README.md

```diff
@@ -1,47 +1,45 @@
 ## Overview
 
-Synopsys Detect is an intelligent scan client that analyzes code in your projects and associated folders to perform compositional analysis. Synopsys Detect can be configured to send scan results to Black Duck, which generates risk analysis when identifying open-source components, licenses, and security vulnerabilities.
+Black Duck Detect is an intelligent scan client that analyzes code in your projects and associated folders to perform compositional analysis. Detect can be configured to send scan results to Black Duck SCA, which generates risk analysis when identifying open-source components, licenses, and security vulnerabilities.
```

Comment:
The SCA looks slightly weird to me here. Should we omit it or say software composition analysis (SCA)? Feel free to ignore if you want it as is.

Reply:
Ya, that is the official term for what was Black Duck, (nee: Black Duck Hub) (They added the SCA to help reduce confusion with the company name.)